### PR TITLE
Summary

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

**Comment by davsclaus** [comment]
Screenshot 2022-03-21 at 17 48 18

**Review by oscerd — APPROVED** [review]

**Comment by orpiske** [comment]
This looks nice!

**Comment by davsclaus** [comment]
The order may seem random, but its the order they are configured .. if we sort them a..z then we may loose a little bit of detail that can help the end user to trouble shoot.

File: core/camel-api/src/main/java/org/apache/camel/spi/PropertiesComponent.java [hunk 1]
```diff
@@ -116,6 +117,25 @@
...
+     * Loads the properties from the default locations and sources filtering them out according to a predicate, and maps
+     * the key using the key mapper.
+     * </p>
+     *
+     * <pre>
+     * {
+     *     &#64;code
+     *     PropertiesComponent pc = getPropertiesComponent();
+     *     Properties props = pc.loadProperties(key -> key.startsWith("camel.component.seda"), StringHelper.dashToCamelCase);
+     * }
```
Comment (essobedo) [inline core/camel-api/src/main/java/org/apache/camel/spi/PropertiesComponent.java:130]:
```suggestion
     *  <code>
     *     PropertiesComponent pc = getPropertiesComponent();
     *     Properties props = pc.loadProperties(key -> key.startsWith("camel.component.seda"), StringHelper.dashToCamelCase);
     * </code>
```
The formatter broke the Javadoc

**Pull request merged**